### DISCUSSION TITLE
Email templates

### DISCUSSION THREAD

**Discussion by Ngomana** [sl-nx/superlogin-next#39] [Q&A] [answered]
Am I suppose to create my own templates, if so:
1. Where should I save them & how should they be saved
2. How should I use them on a basic app

Is there a way to test the application with out configuring email?

**Comment by fynnlyte** [comment, marked as answer]
Template configuration is described in the [README](https://github.com/perfood/couch-auth#email-templates).

Have a look at the [default templates](https://github.com/perfood/couch-auth/tree/master/templates/email) to get started.

You can disable email for getting started:
- [requireEmailConfirm](https://github.com/perfood/couch-auth/blob/master/src/types/config.ts#L91) can be set to false. In that case, no confirmation mail is sent out and you can log in right after signing up.
- There is a [test mode](https://github.com/perfood/couch-auth/blob/master/test/test.config.ts#L8) which disables sending out emails.
